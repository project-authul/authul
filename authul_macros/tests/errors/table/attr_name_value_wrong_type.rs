#[authul_macros::authul_table(name = 42)]
struct Foo;

#[authul_macros::authul_table(name = ())]
struct Bar;

#[authul_macros::authul_table(name = bob)]
struct Baz;

fn main() {
}
