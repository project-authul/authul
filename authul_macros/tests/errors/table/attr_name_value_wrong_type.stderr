error: expected string
 --> tests/errors/table/attr_name_value_wrong_type.rs:1:38
  |
1 | #[authul_macros::authul_table(name = 42)]
  |                                      ^^

error: expected string
 --> tests/errors/table/attr_name_value_wrong_type.rs:4:38
  |
4 | #[authul_macros::authul_table(name = ())]
  |                                      ^^

error: expected string
 --> tests/errors/table/attr_name_value_wrong_type.rs:7:38
  |
7 | #[authul_macros::authul_table(name = bob)]
  |                                      ^^^
